Cognigy/WebchatWidget
# Scroll card with focused button into view

When you have gallery cards with buttons, you will be able to focus the buttons in the cards using tab. Sometimes, when the button in the card receives focus, the card will not be scrolled to the middle (Example: When you have three cards with at least one button each, when a button in card 2 receives focus, the card will not be scrolled to middle). Hence, you will be able to see only half of the focused button and the card in which it is present. 
With this fix, the button inside the card that is focused will be scrolled into view. Additionally, the button groups are assigned aria-labelledby and aria-describedby attributed

To test:
- Create gallery with multiple cards and add at at least one button to each card 
- Navigate using keyboard and see if the focused button is fully visible
